.github/workflows/test-go.yml

```diff
@@ -422,7 +422,7 @@ jobs:
           fi
 
           # If running Go Test 32bit nightly tests, add a flag to rerun failed tests
-          if [[  "${{inputs.name}}" == 'i386' ]]; then
+          if [[  "${{github.repository}}" == 'hashicorp/vault-enterprise' ]]; then
```

Comment:
Can we update the associated comment as we are now widening the scope of the rerun flag

Reply:
🤦 Completely missed that, thanks Steve!

Reply:
Fixed in 8cc4a4f0fe6e111e4d057275e5203f4a7bd33b99